e2e/nomostest/config_sync.go

```diff
@@ -546,6 +547,7 @@ func waitForReconciler(nt *NT, name string) error {
 // RepoSyncClusterRole returns clusterrole with permissions to manage resources in the cluster.
 func RepoSyncClusterRole() *rbacv1.ClusterRole {
 	cr := fake.ClusterRoleObject(core.Name(clusterRoleName))
+	// TODO: make this more granular by test options
```

Comment:
Are you planning to follow up on this yourself?
Can you make a bug to track it?

Reply:
I'm hoping I don't necessarily need to be the person to fix this, but I'll make an issue to track it

Reply:
"put it on the backlog"